src/main/java/uk/gov/hmcts/reform/userprofileapi/config/FeignInterceptorConfiguration.java

```diff
@@ -34,7 +38,7 @@ public RequestInterceptor requestInterceptor(FeignHeaderConfig config) {
                         }
                     }
                 } else {
-                    log.warn("FeignHeadConfiguration", "Failed to get request header!");
+                    log.warn(loggingComponentName, "FeignHeadConfiguration", "Failed to get request header!");
```

Comment:
can you make this consistent by adding {}:: everywhere, as done for other places ? are you updating {}:: for log.info only ?

Reply:
This has been addressed in the following commit: https://github.com/hmcts/rd-user-profile-api/pull/395/commits/776a4b17788086cb843cb412cd6519e17bc09983